src/DateTimeInput/index.jsx

```diff
@@ -426,310 +466,215 @@ export default class DateTimeInput extends Component
         } );
 
         return _.chunk( months, 4 );
-    }
-
-    monthLabel()
-    {
-        const month = $m( this.state.gridStartTimestamp ).month();
-        return copy.months[ month ];
-    }
+    };
 
-    yearLabel()
-    {
-        return $m( this.state.gridStartTimestamp ).year().toString();
-    }
 
-    handleClickCell( { value } )
+    const monthLabel = () =>
```

Comment:
This probably shouldn’t be a function anymore. It can just be:
```
const monthLabel = copy.months[ copy.months[ month ] ];
```

Reply:
done 👍